planning/behavior_path_planner/test/test_behavior_path_planner_node_interface.cpp

```diff
@@ -49,7 +49,6 @@ std::shared_ptr<BehaviorPathPlannerNode> generateNode()
     ament_index_cpp::get_package_share_directory("behavior_path_planner");
 
   std::vector<std::string> module_names;
-  module_names.emplace_back("behavior_path_planner::DynamicAvoidanceModuleManager");
```

Comment:
@zulfaqar-azmi-t4 I guess you don't have to remove this line.

Reply:
[010c69e](https://github.com/autowarefoundation/autoware.universe/pull/5850/commits/010c69ee8748019a15eefd08e47595a9b90ce8d8)